containers/podman-desktop
# feat: add volumes (PVC) to kubernetes

feat: add volumes (PVC) to kubernetes

### What does this PR do?

* Adds PVC to Kubernetes
* Lists in the dashboard, available to edit, delete, etc.

### Screenshot / video of UI



![Screenshot 2024-06-13 at 9 18 00 AM](https://github.com/containers/podman-desktop/assets/6422176/05e6e9b3-c1a1-4fc9-ae9f-47d14c544b88)



https://github.com/containers/podman-desktop/assets/6422176/a438eda6-b41e-46ab-85b2-92f35ffc025e



### What issues does this PR fix or reference?



Closes https://github.com/containers/podman-desktop/issues/7341

### How to test this PR?




Deploy a Kubernetes artifact with a PVC and view / check / play around
with the new dashboard for PVC.

Signed-off-by: Charlie Drage <charlie@charliedrage.com>
